### PR TITLE
 HID: Proper Signal Interrupts for EnableAccelerometer & EnableGyroscopeLow along  with a stub for GetSoundVolume

### PR DESCRIPTION
**Pull request by purpasmart96**
This fixes the freeze when pressing the ocarina on the touch pad in OoT.

### CODE REVIEW

**Comment by archshift** [comment]
Exactly which part of this fixes the bug?

**Comment by purpasmart96** [comment]
For some reason OoT calls the service function  EnableAccelerometer, which has a signal interrupt in it, before that interrupt wasn't being triggered. Also I added bunnei's Gyroscope interrupt event on Init, that should fix issues in other games such as Majora's mask which would let it boot further.

File: src/core/hle/service/hid/hid.cpp [hunk 1]
```diff
@@ -123,6 +124,31 @@ void GetIPCHandles(Service::Interface* self) {
...
+    event_gyroscope->Signal();
+
+    cmd_buff[1] = RESULT_SUCCESS.raw;
+}
+
+void GetSoundVolume(Service::Interface* self) {
+    u32* cmd_buff = Kernel::GetCommandBuffer();
+
+    cmd_buff[1] = RESULT_SUCCESS.raw;
+    cmd_buff[2] = 0x3F; // TODO(purpasmart): Find out if this is the max value for the volume
```
Comment (Subv) [inline src/core/hle/service/hid/hid.cpp]: Please make a variable to hold the current volume level

**Comment by purpasmart96** [comment]
Errrr... I think the name is good enough.

File: src/core/hle/service/hid/hid.cpp [hunk 2]
```diff
@@ -123,6 +124,37 @@ void GetIPCHandles(Service::Interface* self) {
...
+
+    cmd_buff[1] = RESULT_SUCCESS.raw;
+
+    LOG_WARNING(Service_HID, "(STUBBED) called");
+}
+
+void GetSoundVolume(Service::Interface* self) {
+    u32* cmd_buff = Kernel::GetCommandBuffer();
+
+    u8 volume = 0x3F; // TODO(purpasmart): Find out if this is the max value for the volume
```
Comment (bunnei) [inline src/core/hle/service/hid/hid.cpp]: Should be const

**Comment by bunnei** [comment]
LGTM

**Pull request merged**